### ISSUE TITLE
Irradiance dependent noise

### ISSUE THREAD

**Issue by Co1dAt0m** [GuoShi28/CBDNet#30]
Hi, thanks for the great work! 
I have a question about irradiance dependent noise. In the paper 
sigma^2 = L * sigma_s^2 + sigma_c^2, 
however in the source code (https://github.com/GuoShi28/CBDNet/blob/master/utils/AddNoiseMosai.m#L58)
noise_s_map = bsxfun(@times,permute(sigma_s,[3 1 2]),temp_x)
shouldn't sqrt(temp_x) be used? 

Thank you!

**Comment by GuoShi28** [comment]
Yes, I think sqrt(temp_x) is correct and I did not notice this error when coding.  I also suggest you may also need to adjust the range of sigma_s and sigma_c since the releases models are trained using my released code.
